static/js/src/advantage/react/components/Subscriptions/SubscriptionDetails/DetailsContent/DetailsContent.tsx

```diff
@@ -0,0 +1,70 @@
+import { Button, CodeSnippet, Col, Row } from "@canonical/react-components";
```

Comment:
Button is unused but the lint check didn't fail. Is it ignoring tsx files?

Reply:
Ah you're right! I'll fix that in another branch as it looks like there are a lot of ts lint errors (maybe some more config is required for ts files).